example/grind.dart

```diff
@@ -3,21 +3,21 @@
 
 import 'package:grinder/grinder.dart';
 
-main(args) => grind(args);
+Future<dynamic> main(args) => grind(args);
 
 @Task('Initialize stuff.')
-init() {
-  log("Initializing stuff...");
+void init() {
```

Comment:
I like the added use of types on API boundaries; definitely miss the use of types within a method.

Reply:
Yeah, agree